File: 485-max-consecutive-ones/max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int cnt=0,maxcnt=INT_MIN;
        for(int i=0;i<nums.size();i++){
            if(nums[i]==0){
            maxcnt=max(cnt,maxcnt);
            cnt=0;
            }
            else
            cnt++;
        }
        maxcnt=max(cnt,maxcnt);
        return maxcnt;
    }
};

File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int maxCnt=0,cnt=0;
        for(int i=0;i<nums.length;i++)
        {
            if(nums[i]==0)
            {
                maxCnt=Math.max(maxCnt,cnt);
                cnt=0;
            }
            else
                cnt++;
        }
        return Math.max(maxCnt,cnt);
    }
}